From abac8a2068e0333a0f00c276519c24c5c16bedf4 Mon Sep 17 00:00:00 2001
From: Dominic Kempf <dominic.kempf@iwr.uni-heidelberg.de>
Date: Mon, 26 Mar 2018 11:13:42 +0200
Subject: [PATCH] Disable a logging statement that breaks

---
 loopy/kernel/tools.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/loopy/kernel/tools.py b/loopy/kernel/tools.py
index 15840180..cb877eb6 100644
--- a/loopy/kernel/tools.py
+++ b/loopy/kernel/tools.py
@@ -197,11 +197,11 @@ def find_all_insn_inames(kernel):
         assert isinstance(write_deps, frozenset), type(insn)
         assert isinstance(iname_deps, frozenset), type(insn)
 
-        logger.debug("%s: find_all_insn_inames: %s (init): %s - "
-                "read deps: %s - write deps: %s" % (
-                    kernel.name, insn.id, ", ".join(sorted(iname_deps)),
-                    ", ".join(sorted(read_deps)), ", ".join(sorted(write_deps)),
-                    ))
+#         logger.debug("%s: find_all_insn_inames: %s (init): %s - "
+#                 "read deps: %s - write deps: %s" % (
+#                     kernel.name, insn.id, ", ".join(sorted(iname_deps)),
+#                     ", ".join(sorted(read_deps)), ", ".join(sorted(write_deps)),
+#                     ))
 
         insn_id_to_inames[insn.id] = iname_deps
         insn_assignee_inames[insn.id] = write_deps & kernel.all_inames()
-- 
2.11.0

