From 6f2931706f28cd29e3ed72851a7712815a23f474 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ren=C3=A9=20He=C3=9F?= <rene.hess@iwr.uni-heidelberg.de>
Date: Thu, 9 Nov 2017 14:13:34 +0100
Subject: [PATCH] Remove special case for variable in ufl2dot

---
 ufl/formatting/ufl2dot.py | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/ufl/formatting/ufl2dot.py b/ufl/formatting/ufl2dot.py
index 5fdca148..e0387a9b 100644
--- a/ufl/formatting/ufl2dot.py
+++ b/ufl/formatting/ufl2dot.py
@@ -176,13 +176,8 @@ def build_entities(e, nodes, edges, nodeoffset, prefix="", labeller=None):
     if labeller is None:
         labeller = ReprLabeller()

-    # Special-case Variable instances
-    if isinstance(e, Variable):  # FIXME: Is this really necessary?
-        ops = (e._expression,)
-        label = "variable %d" % e._label._count
-    else:
-        ops = e.ufl_operands
-        label = labeller(e)
+    ops = e.ufl_operands
+    label = labeller(e)

     # Create node for parent e
     nodename = "%sn%04d" % (prefix, len(nodes) + nodeoffset)
--
2.11.0
